package.json

```diff
@@ -9,6 +9,8 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@headlessui/react": "^1.7.16",
```

Comment:
❌ 
we already have daisyUI for the tailwind css component, it is not good to add one more css library, can you remove this and build it using daisyUI?

Reply:
Ok now i have changed it and used daisy. I am having a problem with the authentication right now that's why I only uploaded the navba and its translation will wok on the authentication of the navbar in different branch

Reply:
if you are not using it now, can you please remove it?

Reply:
Done